Tasks/Npm/task.json

```diff
@@ -55,6 +60,13 @@
             "type": "filePath"
         },
         {
+            "groupName": "advanced",
+            "name": "verbose",
+            "label": "Verbose logging",
+            "helpMarkDown": "Select to print more information to the console on run",
+            "type": "boolean"
```

Comment:
Since npm has multiple logging levels, it would be nice to offer those options to the user.